config/initializers/permissions.rb

```diff
@@ -130,7 +130,8 @@
                        'work_packages/shares': %i[index]
                      },
                      permissible_on: :project,
-                     require: :member
+                     require: :member,
+                     contract_actions: { shares: %i[index] }
```

Comment:
Should we maybe call this `work_package_shares` instead of just generic `shares`? We should not "pollute" such a generic namespace